examples/text_labels_and_annotations/date.py

```diff
@@ -7,16 +7,22 @@
 formatters.  See :doc:`/gallery/ticks_and_spines/major_minor_demo` for more
 information on controlling major and minor ticks.
 
-All Matplotlib date plotting is done by converting date instances into
-days since 0001-01-01 00:00:00 UTC plus one day (for historical reasons).
-The conversion, tick locating and formatting is done behind the scenes
-so this is most transparent to you.  The :mod:`matplotlib.dates` module
-provides the converter functions `.date2num` and `.num2date`, which convert
-`datetime.datetime` and `numpy.datetime64` objects to and from Matplotlib's
-internal representation.
-
-An alternative way of displaying dates can be seen at
-:doc:`/gallery/ticks_and_spines/date_concise_formatter`.
+Matplotlib date plotting is done by converting date instances into
+days since an epoch (by default 1970-01-01T00:00:00). The
+:mod:`matplotlib.dates` module provides the converter functions `.date2num`
+and `.num2date`, which convert `datetime.datetime` and `numpy.datetime64`
+objects to and from Matplotlib's internal representation.  These data
+types are "registered" with with the unit conversion mechanism described in
```

Comment:
```suggestion
types are registered with with the unit conversion mechanism described in
```

I do not think scare quotes are useful here.